gc/gc-iceberg-inttest/src/intTest/java/org/projectnessie/gc/iceberg/inttest/ITSparkIcebergNessieS3.java

```diff
@@ -90,6 +90,16 @@ protected String warehouseURI() {
     return minio.s3BucketUri(S3_BUCKET_URI).toString();
   }
 
+  @Override
+  protected Map<String, String> sparkHadoop() {
+    Map<String, String> r = new HashMap<>();
+    r.put("fs.s3.impl", "org.apache.hadoop.fs.s3a.S3AFileSystem");
+    r.put("fs.s3a.access.key", minio.accessKey());
+    r.put("fs.s3a.secret.key", minio.secretKey());
+    r.put("fs.s3a.endpoint", minio.s3endpoint());
+    return r;
+  }
```

Comment:
The AWS environment variables set from `nessieParams()` in line 109-112 is not working with spark-3.4. 
Iceberg can understand the env variables. But the error callstack looks like spark is directly trying to access the s3 bucket. Hence, the error. 

Setting the AWS credentials in a good old hadoop configuration for spark. 

Also observed that we have the similar code to set hadoop conf in `ITSparkIcebergNessieCLI` already.

Reply:
BTW the error was that `ITSparkIcebergNessieS3` was failing when spark was trying to access the minIO bucket while creating the namespace.